src/base/operations/create_issuance_request_builder.js

```diff
@@ -73,4 +56,20 @@ export class CreateIssuanceRequestBuilder {
     result.receiver = BaseOperation.balanceIdtoString(request.receiver())
     result.creatorDetails = JSON.parse(request.creatorDetails())
   }
+
+  static _validateOpts (opts) {
```

Comment:
for more transparency, let’s do not extract it to a function. we are preparing it for the back-end guys, let’s make it as easier as possible

Reply:
Updated